index.html

```diff
@@ -122,27 +158,122 @@
             .sizeByClinicalAttribute("Phos", "Result")
             .orderAllTooltipTables(["Source","Location"])
             .enableTrackTooltips(true)
-            .enableZoom(true)
+            .enableZoom(enableAdvanced) //zoom functionality avaiable only in the advance view
+            .advancedView(enableAdvanced)
             .collapseAll();
           timeline();
+          if (enableAdvanced) {
+            $("#advBtn").prop("disabled", true);
+            $("#advBtn").addClass("active");
+            $("#simpleBtn").prop("disabled", false);
+            $("#simpleBtn").removeClass("active");
+            $(".dropdown-toggle").prop("disabled", true);
+            $("#jsonInput").prop("visibility", true);
+          } else {
+            $("#advBtn").prop("disabled", false);
+            $("#advBtn").removeClass("active");
+            $("#simpleBtn").prop("disabled", true);
+            $("#simpleBtn").addClass("active");
+            $(".dropdown-toggle").prop("disabled", false);
+            $("#jsonInput").prop("visibility", false);
+          }
+    }
+
+    function generatePNG(download){
```

Comment:
Try to separate these export functions in a different file. Then call them from index.html accordingly. That way when we move the timeline code to cBioPortal, it will also be easy to call those functions from there. We can't call the functions in `index.html`, because that's not packaged with the code.